overcooked_ai_py/mdp/overcooked_interactive_vpk.py

```diff
@@ -0,0 +1,270 @@
+import pygame
```

Comment:
Again I would probably put in your own project repo, unless it's cleaned up a lot and doesn't require mentions to anything outside of the `overcooked_ai` repo. Same thing about in-depth-reviewing applies here (but again note that I will do more superficial reviews for non-critical code in your project repo).

Reply:
(I had moved this file out of overcooked_ai a while ago, maybe you still had a old version here)